config.tpl.coffee

```diff
@@ -25,7 +25,8 @@ module.exports = (config) ->
 
     # preprocess matching files before serving them to the browser
     # available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
-    preprocessors: %PREPROCESSORS%
+    preprocessors: {%PREPROCESSORS%
+    }
```

Comment:
why this change?

Reply:
to be more consistent with way `files`, `exclude` are built.